CMakeLists.txt

```diff
@@ -174,7 +174,12 @@ check_library_exists(rt shm_open "" LMMS_HAVE_LIBRT)
 
 LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}")
 
-FIND_PACKAGE(Qt5 5.9.0 COMPONENTS Core Gui Widgets Xml Svg REQUIRED)
+IF(LMMS_BUILD_LINUX)
+    FIND_PACKAGE(Qt5 REQUIRED COMPONENTS Core Gui Widgets Xml Svg X11Extras REQUIRED)
```

Comment:
I would appreciate if we find a way to push the x11extras stuff to the vst cmake scripts

Reply:
> I would appreciate if we find a way to push the x11extras stuff to the vst cmake scripts

If you're removing embedding in this PR, why would you still need them?

Reply:
@Rossmaxx on second thought, maybe we keep this... QtX11Info seems to mentioned in this migration guide: https://doc.qt.io/qt-6/extras-changes-qt6.html.

With regards to where this code belongs, we'll likely need this logic for other plugin types (e.g. LV2) so it's most likely in the right place.  We could guard it with a LINUX + VST check if we wanted to be more explicit.

Reply:
x11extras isn't related to the qt5-x11embed submodule we ship right? It should be fine to leave it alone if my understanding is correct.